CONTRIBUTORS.txt

```diff
@@ -114,14 +122,18 @@ needs to be listed here.
 - Norman Maurer <norman_maurer@apple.com>
 - Ole Begemann <ole@oleb.net>
 - OleG <emptystamp@gmail.com>
+- Oleksandr Zhurba <69315809+mob-connection@users.noreply.github.com>
 - Paul Schmiedmayer <paul.schmiedmayer@tum.de>
 - Peter Adams <pp_adams@apple.com>
 - Peter Tolsma <pejato@umich.edu>
 - Petro Rovenskyy <petro.rovenskyy@gmail.com>
+- Philip Brown <122590765+asdf-bro@users.noreply.github.com>
 - Portia <portia-s@users.noreply.github.com>
 - Pushkar N Kulkarni <pushkar.nk@in.ibm.com>
 - Pushparaj Zala <pr92zala@gmail.com>
 - Richard Low <ral@apple.com>
+- Rick Newton-Rogers <104022490+rnro@users.noreply.github.com>
```

Comment:
CC @rnro if you want a real email address here, please submit a pull request with an appropriate entry to the `.mailmap` file. You'll find plenty of examples there, but essentially if you add

```
Your Name <your-real@email.address> <123456+the-weird-github-one@users.noreply.github.com>
```

and then run `scripts/generate_contributors_list.sh` you should appear with proper name & email.